pycqed/instrument_drivers/meta_instrument/LutMans/ro_lutman.py

```diff
@@ -262,105 +324,152 @@ def __init__(self, name, num_res: int=1, **kw):
                            vals=vals.Numbers(min_value=0), unit='s',
                            parameter_class=ManualParameter,
                            initial_value=270e-9)
-
+        self.add_parameter('timeout',
+                           vals=vals.Numbers(min_value=0), unit='s',
+                           parameter_class=ManualParameter,
+                           initial_value=5)
+        self.add_parameter(
+            '_awgs_ro_sequencer_program_expected_hash',
```

Comment:
Given that the UFHQA now takes care of recompiling, this parameter serves no function anymore right?

Reply:
As mentioned before, the LutMan actually specifies the operating mode of the UHFQA. Therefore, it currently keeps track of this parameter. Also, in principle, it would be possible to change the UHFQA configuration behind the back of the RO LutMan. The purpose of these checks is also to ensure that something like that hasn't happened, or if it has, to reconfigure the instrument properly.